### ISSUE TITLE
Add support to auto-reconnect to RabbitMQ if service goes down

### ISSUE THREAD

**Issue by lazyguru** [sublinks/sublinks-api#264]
When the RabbitMQ service is restarted, the backend service gets [forcefully] disconnected. The service should try to handle this gracefully and attempt to reconnect automatically (currently, it just drops the connection and never attempts to reconnect)

## Steps to reproduce
- start docker container for queue
- start backend service
- Login to queue admin (http://localhost:15672/#/connections) and see connection from sublinks
- restart docker container for queue
- Login to queue admin (http://localhost:15672/#/connections) and see no connections

NOTE: It might be easier to do this with the new docker-compose files from https://github.com/sublinks/sublinks-docker/pull/1 so it might be ok to wait on picking this up until that PR is merged and support for the new setup is merged into this repo.
